docs/csharp/language-reference/operators/division-operator.md

```diff
@@ -16,12 +16,16 @@ author: "BillWagner"
 ms.author: "wiwagn"
 ---
 # / Operator (C# Reference)
-The division operator (`/`) divides its first operand by its second operand. All numeric types have predefined division operators.  
+The division operator (`/`) divides its first operand by its second operand. All numeric types have predefined division operators.
   
 ## Remarks  
  User-defined types can overload the `/` operator (see [operator](../../../csharp/language-reference/keywords/operator.md)). An overload of the `/` operator implicitly overloads the [/= operator](division-assignment-operator.md).  
   
- When you divide two integers, the result is always an integer. For example, the result of 7 / 3 is 2. To determine the remainder of 7 / 3, use the remainder operator ([%](../../../csharp/language-reference/operators/modulus-operator.md)). To obtain a quotient as a rational number or fraction, give the dividend or divisor type `float` or type `double`. You can assign the type implicitly if you express the dividend or divisor as a decimal by putting a digit to the right side of the decimal point, as the following example shows.  
+ When you divide two integers, the result is always an integer. For example, the result of 7 / 3 is 2. This is not to be confused with floored division, as the `/` operator rounds towards zero: -7 / 3 is -2.  
+  
+ To obtain a quotient as a rational number, use floating types such as `float` or `double`. This can be done easily by appending `.0` to the end of a number.  
```

Comment:
*floating types* - I'm not sure if that's correct; let's just simplify:
*...use `float` or `double` type. To do that, append `.0` to a number literal as the example below shows.* 

(Append always adds to the end by definition).

Reply:
The suggestion to append `.0` is slightly misleading, that will produce a double literal, you need `.0f` to produce a float literal. It may be better to simply state "use floating point types"  instead of going into details about how to produce a floating point literal.

Reply:
@mikedn @pkulikov What about "Appending `.0` to a number will make it a `double`." ?

Reply:
I'd say that a documentation page about an operator is not the place to explain how a floating point literal is formed. Simply hinting the user to use floating point should be enough.

But then I do not know the intended target audience of this documentation. Something tells me that it may have been intended for beginners, there are too many cases where some things are over simplified to the point that they're wrong while other things are, not sure what word to use for it, accentuated, redundant etc.

I supposed that for beginners it may be worth reminding them how floating literals are formed, many do not realize that there is a difference between 2, 2.0f and 2.0.

Reply:
I think in this case we can omit `float` type mention and only refer to `double` as it's about having a quotient non-integer. OR: remove remark on appending: if both operands are not number literals, that remark is not correct and casting should be used.

Reply:
Good discussion.  I'll suggest not using `.0` at all, as that works on literals, but does not on variables. For the second sentence, I'd say:

> There are many ways to convert between [built in numeric types](../keywords/reference-tables-for-types.md)

Note that I used the internal link in my comment.

Reply:
I think I'll make the first part say ", use the `float`, `double`, or `decimal` types." I was intentionally vague with "floating types" because there is more than just `float` and `double`, as `decimal` exists.